test/e2e/e2e_test.go

```diff
@@ -182,6 +256,84 @@ var _ = Describe("controller", Ordered, func() {
 			EventuallyWithOffset(1, verifyDeploymentReplicas, 2*time.Minute, time.Second).Should(Succeed())
 			By("Verifying we only have one pod left")
 			EventuallyWithOffset(1, verifyOneRunningPod, time.Minute, time.Second).Should(Succeed())
+
+			By("Manually scaling Deployment replicas up")
+			scaleNginxReplicas := func() error {
+				cmd := exec.Command("kubectl", "scale", "deployment",
```

Comment:
can probably do this clientset. get better reporting than forking out to kubectl